src/style.css

```diff
@@ -1,3 +1,103 @@
-body {
+body,
+h1,
+ul,
+li,
+p,
+a {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use the `*` selector as it impacts performance. Instead, set styles only for elements that require them. Zeroing out margins, paddings, or other styles with `*` is inefficient for the browser.